### PR TITLE
Feat/native mode only

### PR DESCRIPTION
**Pull request by envin3**
Only native mode permitted on DAOv2 (Only support Polygon)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dao-v2-ui",
-  "version": "0.1.0",
+  "version": "2.2.0",
```
Comment (oliviera9) [inline package.json]: What motivates skipping 2 majors?

File: src/components/complex/ChainModal/index.jsx [hunk 2]
```diff
@@ -49,7 +49,7 @@ const StyledFaInfoCircle = styled(FaInfoCircle)`
 `
 
 const ChainModal = ({ show, onClose }) => {
-  const activeChainId = useChainId()
+  const activeChainId = useNetwork()
```
Comment (oliviera9) [inline src/components/complex/ChainModal/index.jsx]: Why this change?

File: package.json (hunk 1 above)
Comment (envin3) [reply package.json]: This version was never actually used. This is the dapp v2 so I major is now 2. Minor went from 1 to 2 for the changes.

File: src/components/complex/ChainModal/index.jsx (hunk 2 above)
Comment (envin3) [reply src/components/complex/ChainModal/index.jsx]: This is the correct wagmi hook to use to get connected chain not useChainId()

Comment (oliviera9) [reply src/components/complex/ChainModal/index.jsx]: OK so rename activeChainId into activeNetwork as you are extracting the chain id via .chain.id

File: package.json (hunk 1 above)
Comment (oliviera9) [reply package.json]: This does not comply with semver https://semver.org/.
major is not related to dao v2. minor should be reset if you update major. Have a look at this: https://semver.org/#spec-item-8

Comment (oliviera9) [reply package.json]: Technically this change is a breaking change for a user, thus a major upgrade should be the correct choice. New version should be 1.0.0.

Comment (envin3) [reply package.json]: Right, changed according to specs.

Comment (envin3) [reply package.json]: Updated according to semver.

File: src/components/complex/ChainModal/index.jsx (hunk 2 above)
Comment (envin3) [reply src/components/complex/ChainModal/index.jsx]: done

**Review by oliviera9 — APPROVED** [review]
LGTM
